### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by Nafiz-Ahmed** [community/community#48551] [Copilot Conversations] [answered]
### Select Topic Area

Show & Tell

### Body

Hi, I'm new to copilot. When I trying to install it on VS code it showing this error `[ERROR] [default] [2023-02-25T05:10:54.607Z] GitHub Copilot could not connect to server. Extension activation failed: "Client network socket disconnected before secure TLS connection was established"` I can't understand whats going on. Note: I checked my network connection and it is stable.

**Comment by iamcjwilliams** [comment]
@Nafiz-Ahmed Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Comment by Jonak-Adipta-Kalita** [comment]
I am also having the same issue....

**Reply by ZhangShu126** [reply]
Have you solved it

**Reply by Jonak-Adipta-Kalita** [reply]
Yup, Check my Bottom Comment

**Reply by Jonak-Adipta-Kalita** [reply]
https://github.com/orgs/community/discussions/48551#discussioncomment-6081279

**Comment by ZhangShu126** [comment]
I am also having the same issue....

**Comment by Jonak-Adipta-Kalita** [comment, marked as answer]
I was having same issue, but when I switched to a better wifi and restarted my pc it works now
